webapp/components/reftest-analyzer.js

```diff
@@ -52,12 +55,18 @@ class ReftestAnalyzer extends LoadingState(PolymerElement) {
         }
       </style>
 
-      <div id='zoom'>
+      <div id="zoom">
         <svg xmlns="http://www.w3.org/2000/svg" shape-rendering="optimizeSpeed">
           <g id="zoomed">
             <rect width="250" height="250" fill="white"/>
           </g>
         </svg>
+
+        <div id="info">
+          <strong>Pixel at:</strong> [[curX]], [[curY]] <br>
+          <strong>Image before:</strong> [[getRGB(canvasBefore, curX, curY)]] <br>
```

Comment:
Can we change this to "Test" and "Reference" or something, and eventually the test URLs?

Reply:
We should change the display text along with the DOM identifiers together to avoid confusion. Besides, I'll need to double check whether "before" is always guaranteed to be the test image. I'll do it in another PR.